src/components/seasons/SuccessSeasonDrawer.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Drawer, Button } from "@mui/material";
+import IconModalSeason from '../../../assets/icons/season.svg';
+
+
+interface SuccessDrawerProps {
+  open: boolean;
+  dismiss: () => void;
+}
+
+const SuccessDrawer: React.FC<SuccessDrawerProps> = ({ open, dismiss }) => {
+  return (
+    <Drawer anchor="right" open={open} onClose={dismiss}>
+      <div
```

Comment:
If this confirmation Drawer is similar in more features we should create a single component